static/app/views/starfish/views/spans/spanSummaryPanel/index.tsx

```diff
@@ -37,28 +41,42 @@ export function SpanSummaryPanel({span, onClose}: Props) {
       <Header>{t('Span Summary')}</Header>
 
       <BlockContainer>
-        <Block title={t('First Seen')}>
+        <Block
+          title={t('First Seen')}
+          description={t(
+            'The first time this span was ever seen in the current retention window'
+          )}
+        >
           <TimeSince date={spanMetrics?.first_seen} />
           {firstSeenSpanEvent?.release && (
             <ReleasePreview release={firstSeenSpanEvent?.release} />
           )}
         </Block>
 
-        <Block title={t('Last Seen')}>
+        <Block
+          title={t('Last Seen')}
+          description={t('The most recent time this span was seen')}
+        >
           <TimeSince date={spanMetrics?.last_seen} />
           {lastSeenSpanEvent?.release && (
             <ReleasePreview release={lastSeenSpanEvent?.release} />
           )}
         </Block>
 
-        <Block title={t('Total Spans')}>{spanMetrics?.count}</Block>
+        <Block
+          title={t('Total Spans')}
+          description={t('The total number of times this span was seen in all time')}
+        >
+          {spanMetrics?.count}
+        </Block>
 
-        <Block title={t('Total Time')}>
-          <Duration
-            seconds={spanMetrics?.total_time / 1000}
-            fixedDigits={2}
-            abbreviation
-          />
+        <Block
+          title={t('App Impact')}
+          description={t(
+            'The total exclusive time taken up by this span vs. entire application'
```

Comment:
Should we say "exclusive" time here? I have a feeling this is just grounds for more confusion